package.json

```diff
@@ -17,7 +17,7 @@
     "bcrypt": "^5.1.1",
     "dotenv": "^16.4.0",
     "framer-motion": "^10.18.0",
-    "jsonwebtoken": "^9.0.2",
```

Comment:
@Akib-Vijapura  we are using jose instead of jsonwebtoken now, so removed jsonwebtoken